app/src/main/java/org/mozilla/fenix/ext/BookmarkNode.kt

```diff
@@ -12,19 +12,18 @@ import org.mozilla.fenix.R
 
 var rootTitles: Map<String, String> = emptyMap()
 
-@SuppressWarnings("ReturnCount")
+@SuppressWarnings("ReturnCount", "ComplexMethod")
 suspend fun BookmarkNode?.withOptionalDesktopFolders(
-    context: Context?
+    context: Context?,
+    showMobileRoot: Boolean = false
 ): BookmarkNode? {
     // No-op if node is missing.
-    if (this == null) {
-        return null
-    }
+    if (this == null) return null
```

Comment:
Let's be consistent with @jhugman's feedback [here](https://github.com/mozilla-mobile/fenix/pull/3398/files#diff-f4e9cb78eb3bbc10057cf15de6b220e1R96) and do `{ return null }`

Reply:
I can't see the actual feedback, but I can revert to using braces and newlines. @jhugman's suggestion seems to violate the Android Kotlin Style Guide, which I was following. https://developer.android.com/kotlin/style-guide#braces